Add NEWS.md entry about image geom functionality (#13)
funkyheatmap/funkyheatmap: NEWS.md

> # funkyheatmap 0.3.1
> 
> ## MAJOR CHANGES
> 
> * Plot images as a geom by specifying either a path, or a directory and an extension. Not yet available on CRAN.

Reviewer: If you want to mention that funkyheatmap is not on CRAN yet, I think the header is the right place ^^

```suggestion
# funkyheatmap 0.3.1 (unreleased)

## MAJOR CHANGES

* Plot images as a geom by specifying either a path, or a directory and an extension.
```
